### ISSUE TITLE
error on Drupal\Core\Entity\EntityViewBuilder::viewField() must implement interface Drupal\Core\Field\FieldItemListInterface, null given

### ISSUE THREAD

**Issue by sharique** [drunomics/swiper-gallery#29]
Drupal - 9.1.13
swiper_gallery - 2.1.1

I 've images file in product variation entity from Drupal commerce.

```The website encountered an unexpected error. Please try again later.
TypeError: Argument 1 passed to Drupal\Core\Entity\EntityViewBuilder::viewField() must implement interface Drupal\Core\Field\FieldItemListInterface, null given, called in /var/www/html/docroot/modules/contrib/swiper_gallery/src/Plugin/Field/FieldFormatter/SwiperGalleryFormatter.php on line 432 in Drupal\Core\Entity\EntityViewBuilder->viewField() (line 447 of core/lib/Drupal/Core/Entity/EntityViewBuilder.php).```

**Comment by mbm80** [comment]
added a setting for the fieldname
https://github.com/drunomics/swiper-gallery/pull/30

let me know if this works for you

**Comment by sharique** [comment]
Thanks, for quick update.  
Please posting this module to Drupal.org, so that it is easy to find and add to project.

**Comment by mbm80** [comment]
merged & created 2.1.2

> Please posting this module to Drupal.org

we can do that in the upcoming weeks or so
